cmd/eksctl-anywhere/cmd/copypackages.go

```diff
@@ -144,3 +181,56 @@ func (c CopyPackagesCommand) copyImages(ctx context.Context, dstRegistry registr
 	}
 	return nil
 }
+
```

Comment:
I had a PR out to add tags to the bundle, that was the goal to have digest and version in both bundles.

Reply:
is that something we should resurrect?  Im wondering what the lift is there?

Reply:
talked on slack, we can take that as a follow up since its a bigger change to packages.